tests/test_penv.py

```
@@ -16,63 +16,71 @@
+    assert os.path.isfile(uv_exe), f"uv executable not found at {uv_exe}"
+    # Test that uv works by running 'uv help'
+    try:
+        result = subprocess.run(
+            [uv_exe, "help"], capture_output=True, text=True, check=True, timeout=10
+        )
+        print("\nUV Help Output:")
+        print(result.stdout)
+        assert "uv" in result.stdout.lower()
+        assert "help" in result.stdout.lower() or "usage" in result.stdout.lower()
+    except subprocess.CalledProcessError as e:
+        raise AssertionError(f"uv help command failed: {e}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)